tests/test_cli_studio.py

```diff
@@ -440,3 +440,60 @@ def test_studio_run(capsys, mocker, tmp_dir):
         "priority": 5,
         "compute_cluster_name": "default",
     }
+
+
+def test_studio_run_non_zero_exit_code(capsys, mocker, tmp_dir):
```

Comment:
we can parametrize and test FAILED, CANCELED with different exit codes - 1 and 2